info/v1/machine.go

```diff
@@ -222,6 +222,9 @@ type MemoryInfo struct {
 }
 
 type NVMInfo struct {
+	// The total NVM capacity in bytes for memory mode.
+	MemoryModeCapcity uint `json:"memory_mode_capcity"`
```

Comment:
probably want a uint64 here?

Reply:
I may have missed this in the power budget below as well... do you know if there is a range of possible values for that?

Reply:
Of course uint64 is a better choice for memory capacity. I corrected this.
power budget is in watts and I think that this value does not exceed uint range but I added information about unit of power budget in the comment.